src/libs/actions/IOU.ts

```diff
@@ -3700,7 +3700,14 @@ function requestMoney(requestMoneyInformation: RequestMoneyInformation) {
     }
 
     InteractionManager.runAfterInteractions(() => TransactionEdit.removeDraftTransaction(CONST.IOU.OPTIMISTIC_TRANSACTION_ID));
+
     Navigation.dismissModal(isSearchTopmostCentralPane() ? undefined : activeReportID);
+
+    const trackReport = Navigation.getPreviousTrackReport(linkedTrackedExpenseReportAction?.childReportID);
+    if (trackReport?.key) {
+        Navigation.isNavigationReady().then(() => Navigation.removeScreenByKey(trackReport.key));
```

Comment:
We're calling `isNavigationReady()` twice (first from here and then from inside the `removeScreenByKey` method itself). Is this intentional?